### PR TITLE
Update silkscreen and CHANGELOG for REV F 

### LINKED ISSUE #233: AMDC: Silkscreen label for "REV" marker is incorrect on REV F
The silkscreen label on the AMDC which labels which revision the PCB is was not updated on REV F... somehow slipped through the design review process. Could have been caught with appropriate check lists :)

This should be resolved in the Altium files and new gerbers/output files should be uploaded to the GitHub `REVxxxF` folder.

--- 

Anyway, we solved it on the new boards by just putting a label on the board over the incorrect string with the correct one: "REV20231005F"

### LINKED ISSUE #234: AMDC: Update CHANGELOG file for REV F hardware
We forgot to update the CHANGELOG.md file with the new hardware, see here:

https://github.com/Severson-Group/AMDC-Hardware/blob/develop/CHANGELOG.md

### PR DESCRIPTION
**Pull request by codecubepi**
## Closes #233 

See the fix below:

image

## Closes #234 

CHANGELOG.md has been updated for REV F, listing the four fixes originally addressed by @DivyaMendpara :

> ## [REV20231005F](https://github.com/Severson-Group/AMDC-Hardware/milestone/8)
> 
> - Change Screw 1 ... 13 P/N to `36-9900-ND` ([#189](https://github.com/Severson-Group/AMDC-Hardware/issues/189))
> - Fix PCB Footprint `TSOP65P640X120-16` ([#211](https://github.com/Severson-Group/AMDC-Hardware/issues/211))
> - Add termination resistors for incoming GPIO signals ([#226](https://github.com/Severson-Group/AMDC-Hardware/issues/226))
> - Increase R97 pull-down strength to fix PWM gate signals during boot-up ([#227](https://github.com/Severson-Group/AMDC-Hardware/issues/227))

### CODE REVIEW

File: REV20231005F/PCB_3D_Bottom.PNG
Comment (npetersen2) [inline REV20231005F/PCB_3D_Bottom.PNG:1]: This is a rather pedantic comment, but can you recapture the screenshot with a white background? This is more typical (see the image you are replacing). You might have to alter Altium's View settings to change to white.

File: REV20231005F/PCB_3D_Top.PNG
Comment (npetersen2) [inline REV20231005F/PCB_3D_Top.PNG:1]: This is a rather pedantic comment, but can you recapture the screenshot with a white background? This is more typical (see the image you are replacing). You might have to alter Altium's View settings to change to white.

**Review by npetersen2 — CHANGES_REQUESTED** [review]
Thanks @codecubepi ! 

Funny that when generating the `odb` files, there are numerous changes to many files, but the `gerber` files have the expected change of only the `GTO` file... after reviewing  the odb changes, I think they are not functional changes, just formating of the file. Maybe due to you using Altium 24 and the REV F files were originally generated using 23.9.

Overall, looks great! Just a small request below:

File: REV20231005F/PCB_3D_Bottom.PNG
Comment (npetersen2) [reply REV20231005F/PCB_3D_Bottom.PNG:1]: While you're at it, would be really nice to recapture the REV E images to replace the docs page..it have a black background currently:

https://docs.amdc.dev/hardware/revisions/rev-e/

**Comment by codecubepi** [comment]
> Funny that when generating the `odb` files, there are numerous changes to many files, but the `gerber` files have the expected change of only the `GTO` file... after reviewing the odb changes, I think they are not functional changes, just formating of the file. Maybe due to you using Altium 24 and the REV F files were originally generated using 23.9.

Well for the `gerber` files, I only regenerated and replaced the `GTO` file, since I figured that should be the only one that changed. I was not sure how to do the same with the `odb` files, but yes, it appears that regenerating all of them only changed a few.

File: REV20231005F/PCB_3D_Bottom.PNG
Comment (codecubepi) [reply REV20231005F/PCB_3D_Bottom.PNG:1]: Done

File: REV20231005F/PCB_3D_Top.PNG
Comment (codecubepi) [reply REV20231005F/PCB_3D_Top.PNG:1]: Done.

**Comment by codecubepi** [comment]
@npetersen2 

This is ready for a re-review. I'll use the new white background PCB captures in the docs.

**Review by npetersen2 — APPROVED** [review]
Looks great, thanks @codecubepi !

Due to commit history, let's squash and merge this PR (or you can rebase and squash locally into 1 commit, then do a normal merge commit on GitHub)

**Pull request merged**